--- /etc/xdg/awesome/rc.lua	2015-09-01 07:28:43.000000000 -0400
+++ /home/work/.config/awesome/rc.lua	2016-02-05 16:34:38.527188593 -0500
@@ -41,7 +41,8 @@
 
 -- {{{ Variable definitions
 -- Themes define colours, icons, font and wallpapers.
-beautiful.init("/usr/share/awesome/themes/default/theme.lua")
+--beautiful.init("/usr/share/awesome/themes/default/theme.lua")
+beautiful.init(awful.util.getdir("config") .. "/themes/my_theme/theme.lua")
 
 -- This is used later as the default terminal and editor to run.
 terminal = "x-terminal-emulator"
